cityofaustin/atd-vz-data
# Crash details page: header

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17181

## Testing
**URL to test:** 
Locally

**Steps to test:**

1. Check out this branch
2. Follow [the steps](https://github.com/cityofaustin/atd-vz-data/pull/1440) to start the data model collector branch and run the CRIS import ETL
3. Review the code, see that the header data is now pulling from the new crashes table
4. In the VZE click on diff crashes and make sure the headers are rendering correctly


---
#### Ship list
